### PR TITLE
Book details

### PR DESCRIPTION
**Pull request by cvega08**
## Summary

I did the view of details of the book, the request to the api with the id, also the action and mutation for the store.

##Screen Shot
details

##Trello Card
https://trello.com/c/pz70dnjJ

### CODE REVIEW

File: vue/wolox-books/src/views/Book.vue [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+          h1.title
+           | {{ selectedBook.title }}
+          span.genre
+           | ({{ selectedBook.genre }})
+        .other-details
+          h4.item
+            | {{ detailsBook.author }}:
+          span.value
+            | {{ selectedBook.author }}
+          br
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Book.vue]: avoid using br, what can you do to get the same new line?

File: vue/wolox-books/src/views/Book.vue [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+          h4.item
+            | {{ detailsBook.publisher }}:
+          span.value
+            | {{ selectedBook.publisher }}
+          br
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Book.vue]: same

File: vue/wolox-books/src/views/Home.vue [hunk 3]
```diff
@@ -4,11 +4,16 @@
...
         img.wolox-icon-small(src='../assets/wolox_logo.svg' alt='Wolox logo')
         p.icon-title
-          | BOOKS
+          | {{ labels.title }}
       button.logout(@click='logout()')
-        | Logout
+        | {{ labels.logout }}
     .books-container
-      .book(v-for='book of books' :key='book.id')
+      router-link(
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Home.vue]: `router-link.book`?

File: vue/wolox-books/src/views/Home.vue [hunk 4]
```diff
@@ -4,11 +4,16 @@
...
+        v-for='book of books'
+        :to='{ path: `${routes.book}/${book.id}`}'
+        :key='book.id'
+        class='book'
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Home.vue]: delete this line

File: vue/wolox-books/src/views/Home.vue (hunk 3 above)
Comment (cvega08) [reply vue/wolox-books/src/views/Home.vue]: You are right.

**Pull request merged**